src/components/table/sort.ts

```diff
@@ -9,11 +9,20 @@ export function descendingComparator<T>(a: T, b: T, orderBy: keyof T): number {
   const bValue = b[orderBy] ?? '';
   const aValue = a[orderBy] ?? '';
 
-  if (bValue < aValue) {
+  // blank values at the end always (any order)
+  if (aValue === '' || aValue === null) {
+    return 1;
+  }
+
+  if (bValue === '' || bValue === null) {
     return -1;
   }
+
+  if (bValue < aValue) {
+    return order === 'desc' ? -1 : 1;
```

Comment:
nice, we may need to check 'desc' for aValue/bValue empty state as well and return -1/1 accordingly.

Reply:
actually nvm, i realize we want this at the bottom, no need to check for `desc`.. my bad